drivers/openstack/openstack.go

```diff
@@ -44,6 +44,7 @@ type Driver struct {
 	FloatingIpPoolId string
 	IpVersion        int
 	client           Client
+	PrivateIPAddress string
```

Comment:
Yes, I see. but in my opinion, the machine need its private ip no matter whether set assignFloatingIP or not, so it should has another attribute.